src/Anime.java

```diff
@@ -0,0 +1,60 @@
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class Anime {
+	private final String title;
+	private final float rating;
+	private final Genre genre;
+	private final LocalDate releasedDate;
+
+	public Anime(String title, float rating, Genre genre, LocalDate releasedDate) {
+		this.title = title;
+		this.genre = genre;
+		this.releasedDate = releasedDate;
+		if (rating >= 0 && rating <= 5) {
```

Comment:
```suggestion
		if (0 <= rating && rating <= 5) {
```

このようにしましょう。
理由も考えてみてほしいです！

Reply:
あとリンさんに確認してほしいのですがレーティングが0というのはありえるのでしょうか？